keyboards/lets_split/keymaps/OLED_sample/keymap.c

```diff
@@ -2,7 +2,7 @@
 #include "bootloader.h"
 #include "action_layer.h"
 #include "eeconfig.h"
-#include "tmk_core/protocol/lufa/LUFA-git/LUFA/Drivers/Peripheral/TWI.h"
+#include "lib/lufa/LUFA/Drivers/Peripheral/TWI.h"
```

Comment:
I think that `lib/lufa/LUFA/` is not required here, as that breaks the `LUFA_PATH` support.

But I'm not sure if it should be 
```c
#include "Drivers/Peripheral/TWI.h"
// Or
#include "LUFA/Drivers/Peripheral/TWI.h"
```